examples/gallery/main.rs

```diff
@@ -8,21 +8,39 @@ use wasm_bindgen::prelude::*;
 
 slint::include_modules!();
 
+#[cfg(target_arch = "wasm32")]
+#[wasm_bindgen]
+pub fn load_font_from_bytes(font_data: &[u8]) -> Result<(), JsValue> {
+    slint::register_font_from_memory(font_data.to_vec());
+    Ok(())
+}
+
 use std::rc::Rc;
 
 use slint::{Model, ModelExt, ModelRc, SharedString, StandardListViewItem, VecModel};
 
-#[cfg_attr(target_arch = "wasm32", wasm_bindgen(start))]
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
 pub fn main() {
     // This provides better error messages in debug mode.
     // It's disabled in release mode so it doesn't bloat up the file size.
     #[cfg(all(debug_assertions, target_arch = "wasm32"))]
     console_error_panic_hook::set_once();
 
+    // For native builds, initialize gettext translations
+    #[cfg(not(target_arch = "wasm32"))]
     slint::init_translations!(concat!(env!("CARGO_MANIFEST_DIR"), "/lang/"));
 
     let app = App::new().unwrap();
 
+    // For WASM builds, select translation after App::new()
```

Comment:
Probably something we should do in i_slint_core instead.

https://github.com/slint-ui/slint/blob/783f7403b13b23e76343fd2576430b94929d57ce/internal/core/translations.rs#L375

Actually, we're already using the `sys_locale` crate and it should support wasm. But we probably need to enable extra feature in it for it to work?